code_of_conduct.md

```diff
@@ -1 +1,116 @@
 # Code of Conduct
+
+## ND in Tech Mission Statement
+
+Neurodiversity In Tech (ND in Tech, for short) is an online community dedicated to supporting neurodivergent professionals in the technology industry. Our mission is to enable members to network, navigate their careers, and empower each other while advocating for meaningful social change. ND in Tech serves as a crucial hub for sharing experiences, offering support, and promoting inclusivity within the tech world. We aim to create a welcoming environment where diversity is celebrated and every individual feels valued.
+
+## Scope of This Code of Conduct
+
+This code of conduct applies to interactions between members of this community within the spaces that ND in Tech hosts.
+
+### In Scope
+
+- Inside of ND in Tech's Slack
+- At ND in Tech events - virtual or in person
+
+### Out of Scope
+
+There are situations outside of the control of ND in Tech representatives. We make no representation that we can enforce this code of conduct in these scenarios:
+
+- Interactions on other online platforms not administered by ND in Tech representatives.
+- In-person meet-ups not sponsored by ND in Tech.
+
+We will consider *any* harassing behavior discovered by ND in Tech representative as a matter that could affect the health of the community and take appropriate actions as listed in the Grievance Review Process.
+
+## Community Membership/Inclusion
+
+Membership is for neurodivergent professionals in technology or those who support such individuals in a professional capacity. A medical diagnosis is not needed to join the community.
+
+## Expected Behavior
+
+- Bring your authentic self.
+- Treat all members with dignity and respect.
+- Support and uplift members of the community.
+- [Assume positive intent](https://www.psychologytoday.com/us/blog/mind-the-manager/202203/when-in-doubt-assume-positive-intent).
+- Keep conversation roughly in line with the subject of the channels in Slack - there are plenty of channels that cover a variety of conversational topics.
+  - Channels prefixed with **#community-** focus on topics important to a specific demographic.
+  - Channels prefixed with **#interest-** are for discussions around professional development, hobbies, etc.
+  - Channels prefixed with **#flavor-** or **#challenge-** focus around advice-seeking and outlets for health-related discussions.
+  - Channels prefixed with **#tool-** focus on solutions or strategies to address specific needs.
+  - Channels prefixed with **#intersection-** are private and request-to-join only.
```

Comment:
Maybe add something about the `intersection` channels for being in specific marginalized communities? Looks great though!

Reply:
>...request-to-join only

@hayleyw7 These have zero visibility in Slack if you're not in them - how do people know what ones there are to request an invite?

Reply:
@hayleyw7 bump

Reply:
@royemosby There's a list pinned in #announcements, the putting that same paragraph here is probably useful.

"#intersection channels are private & cater to different marginalized identities: queer, nonbinary, trans, women, & marginalized genders (women+trans). DM me or another member for an invite, & feel free to create new ones to represent your own intersectionalities."

Reply:
Moved the Slack channel details to a linked Google Doc